android/src/main/java/io/invertase/googlemobileads/ReactNativeGoogleMobileAdsCommon.java

```diff
@@ -30,13 +30,15 @@
 import com.google.android.gms.ads.AdRequest;
 import com.google.android.gms.ads.AdSize;
 import com.google.android.gms.ads.admanager.AdManagerAdRequest;
+import io.invertase.googlemobileads.common.ReactNativeAdView;
 import io.invertase.googlemobileads.common.ReactNativeEventEmitter;
 import java.util.ArrayList;
 import java.util.Map;
 import java.util.Objects;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 import javax.annotation.Nullable;
+import android.util.Log;
```

Comment:
this import was added but no lines reference it, likely vestige of implementation debugging